ui-spacetimechart/src/components/SpaceTimeChart.tsx

```diff
@@ -154,7 +157,11 @@ export const SpaceTimeChart = (props: SpaceTimeChartProps) => {
       hideGrid: !!hideGrid,
       hidePathsLabels: !!hidePathsLabels,
       showTicks: !!showTicks,
+      hideDates: !!hideDates,
       theme: fullTheme,
+      captionSize: hideDates
```

Comment:
`captionSize` refers more to `captionHeight` no ?

Reply:
It's `captionSize`, because it's `captionHeight` most of the time, but it's also `captionWidth` when `swapAxis` is true.